spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSQLContext.scala

```diff
@@ -37,6 +42,123 @@ class CassandraSQLContext(sc: SparkContext) extends SQLContext(sc) {
   @transient
   val conf = sc.getConf
 
+  @transient
+  private val clusterReadConf = mutable.Map[String, ReadConf]()
+  @transient
+  private val keyspaceReadConf = mutable.Map[Seq[String], ReadConf]()
+  @transient
+  private val tableReadConf = mutable.Map[Seq[String], ReadConf]()
+
+  /** Add table level read configuration settings */
+  def addTableReadConf(keyspace: String, table: String, conf: SparkConf, cluster: Option[String]) = {
```

Comment:
I must have missed one thing during previous review - I'd really like to pass `ReadConf` by all `addXXXReadConf` methods, `WriteConf` by all `addXXXWriteConf` methods and `CassandraConnectorConf` by `addClusterLevelConnConf` method.